### PR TITLE
raindrops: add exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by kytrinyx — APPROVED** [review]

**Comment by glennj** [comment]
Should I wait until this is merged before creating a PR to avoid merge conflicts in config.json?

**Comment by ErikSchierboom** [comment]
@glennj Merged!

**Pull request merged**
